src/lavinmq/user_store.cr

```diff
@@ -119,16 +119,19 @@ module LavinMQ
           @replicator.register_file f
         end
       else
-        tags = [Tag::Administrator]
         Log.debug { "Loading default users" }
-        create("guest", "guest", tags, save: false)
-        add_permission("guest", "/", /.*/, /.*/, /.*/)
-        save!
+        create_default_user
       end
       create_direct_user
       Log.debug { "#{size} users loaded" }
     end
 
+    private def create_default_user
+      add(Config.instance.default_user, Config.instance.default_password, "SHA256", tags: [Tag::Administrator], save: false)
```

Comment:
Before we used `create` now `add` what's the difference?

Reply:
add takes `password_hash` and create takes `password`. Besides that, they are very similar.

Reply:
Ok